core/src/test/java/com/microsoft/applicationinsights/internal/config/ValidProcessorWithIncludeExcludeType.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.applicationinsights.internal.config;
+
+import com.microsoft.applicationinsights.extensibility.TelemetryProcessor;
+import com.microsoft.applicationinsights.telemetry.Telemetry;
+
+public class ValidProcessorWithIncludeExcludeType implements TelemetryProcessor {
```

Comment:
Why do you need this as a public class? Not sure what it does

Reply:
This should technically be a part of the test package. Its a stub class which supports testing. I will double check if by mistake the was not was not the part of the test package and move it.